index.html

```diff
@@ -54,5 +56,249 @@ <h1 class="name">
             </div>
     </div>
 </section>
+
+<!-- project two -->
+<section class="box">
+    <div  class="workCard1"> 
+    <div class="block1">
+        <img src="images/work1.png" alt="">

```

Comment:
- [ ] I will implore you to use this from the CSS as decorations are always backgrounded images, not img tags and this image is a mere decoration.